coder_sniffer/Drupal/Sniffs/Arrays/ArraySniff.php

```diff
@@ -254,10 +254,22 @@ public function process(File $phpcsFile, $stackPtr)
                 break;
             }
 
+            // Define tokens that require extra indentation when found at the start of a line.
+            // Expects: '->', '?', ':'.
+            $extraIndentTokens = [
+                T_OBJECT_OPERATOR,
+                T_INLINE_THEN,
+                T_INLINE_ELSE,
+            ];
+
+            // Determine whether extra indentation required for the next array line.
+            // True where the line starts with one of the defined extra indent tokens.
+            $extraIndentRequired = (in_array($tokens[$newLineStart]['code'], $extraIndentTokens, true));
```

Comment:
the extra true parameter should not be necessary, but also does not hurt.

The coding standard error was related to the if(), where you need to compare the result of in_array() with a bool.

But this style is also perfectly fine.

Reply:
That makes more sense, i must have misread the sniff error.

Assume the following would not be flagged as error:

````php
if (in_array($tokens[$newLineStart]['code'], $increasedNestingTokens, true) === true) {
   ...
}
````

Either way, thanks for merging!